pilot/pkg/networking/core/v1alpha3/cluster_builder_test.go

```diff
@@ -1698,7 +1713,7 @@ func TestConcurrentBuildLocalityLbEndpoints(t *testing.T) {
 			Service:     service,
 			ServicePort: servicePort,
 			Endpoint: &model.IstioEndpoint{
-				Address:      "192.168.1.4",
+				Addresses:    []string{"192.168.1.4"},
```

Comment:
TestConcurrentBuildLocalityLbEndpoints requires dual-stack tests

Reply:
done, already add unit test for `workload-1` in `TestConcurrentBuildLocalityLbEndpoints`